File: Server/src/commonStructures/networkPlayerGameStateStream.ts
import NetworkPlayerSimulationState from "./networkPlayerSimulationState";

export default class NetworkPlayerGameStateStream {
    constructor(id:number,simulationState:NetworkPlayerSimulationState) {
        this.id = id;
        this.simulationState = simulationState;
    }
    public id:number;
    public simulationState:NetworkPlayerSimulationState;
}

File: Assets/Scripts/Network/CommonStructures/NetworkPlayerGameStateStream.cs
namespace Network
{
    /// <summary>
    /// The game state of a given player.
    /// </summary>
    public class NetworkPlayerGameStateStream
    {
        /// <summary>
        /// The id of the player.
        /// </summary>
        public int id;
        /// <summary>
        /// The simulation state of the player.
        /// </summary>
        public NetworkPlayerSimulationState simulationState;
    }
}
